src/scrapers/leumi-card.js

```diff
@@ -288,12 +288,89 @@ async function fetchTransactions(browser, options) {
   return allResults;
 }
 
+async function fetchAccountsData(browser) {
+  function parseFloatValue(value) {
+    return parseFloat(value.replace(',', ''));
+  }
+
+  const homePageUrl = `${BASE_URL}/Registred/HomePage.aspx`;
+  const page = await browser.newPage();
+  await page.goto(homePageUrl);
+
+  const results = {};
+  await clickButton(page, 'a#PlaceHolderMain_HomePage1_HomePageTop1_lnkShowListDisplay');
+  await page.waitForNavigation({ waitUntil: 'load' });
+
+  const cardsContainers = await page.$$('.newCreditCard_bg');
+  for (let cardIndex = 0; cardIndex < cardsContainers.length; cardIndex += 1) {
+    const cardContainer = cardsContainers[cardIndex];
+
+    const cardTop = await cardContainer.$('.newCreditCard_top');
+    const cardNameContainer = await cardTop.$('.creditCard_name');
+    const cardNameDetails = await cardNameContainer.$$('li');
+    const cardName = await page.evaluate((li) => {
+      return li.innerText;
+    }, cardNameDetails[0]);
+    const cardNumber = await page.evaluate((li) => {
+      return li.innerText.replace('(', '').replace(')', '');
+    }, cardNameDetails[1]);
+
+    const cardMiddle = await cardContainer.$('.newCreditCard_listInfo');
+    const cardCharges = await cardMiddle.$$('ul');
+    const cardLocalCharges = await cardCharges[0].$$('li');
+    const cardLocalChargeSpans = await cardLocalCharges[0].$$('span');
+    const chargedDate = await page.evaluate((span) => {
+      return span.innerText.replace('(', '').replace(')', '').trim();
+    }, cardLocalChargeSpans[1]);
+    let chargedDayOfMonth = null;
+    if (chargedDate.length > 0) {
+      chargedDayOfMonth = parseInt(chargedDate.split('/')[0], 10);
+    }
+    let upcomingCardLocalCharge = await page.evaluate((span, SHEKEL_CURRENCY_SYMBOL) => {
+      return span.innerText.replace(SHEKEL_CURRENCY_SYMBOL, '').trim();
+    }, cardLocalChargeSpans[0], SHEKEL_CURRENCY_SYMBOL);
+    upcomingCardLocalCharge = parseFloatValue(upcomingCardLocalCharge);
+
+    const cardForeignCharges = await cardCharges[1].$$('li');
+    const cardForeignChargeSpans = await cardForeignCharges[0].$$('span');
+    let upcomingCardForeignChargeInILS = await page.evaluate((span, SHEKEL_CURRENCY_SYMBOL) => {
+      return span.innerText.replace(SHEKEL_CURRENCY_SYMBOL, '').trim();
+    }, cardForeignChargeSpans[0], SHEKEL_CURRENCY_SYMBOL);
+    upcomingCardForeignChargeInILS = parseFloatValue(upcomingCardForeignChargeInILS);
+
+    const cardBottom = await cardContainer.$('.creditFrame_width');
+    const creditUtilizationContainer = await cardBottom.$('a');
+    let creditUtilization = await page.evaluate((a, SHEKEL_CURRENCY_SYMBOL) => {
+      return a.innerText.replace(SHEKEL_CURRENCY_SYMBOL, '').trim();
+    }, creditUtilizationContainer, SHEKEL_CURRENCY_SYMBOL);
+    creditUtilization = parseFloatValue(creditUtilization);
```

Comment:
@eshaham you are probably right. I am just not sure what your suggested value is giving us, since it basically the `balance` and the `upcomingCardLocalCharge` properties, unlike this value which is, in other words, how much left for me to spend, which is much more effective.
In #103 I guess we will just stick with `creditLimitUtilizationAmount`, but I think we should decide once and for all the other scrapers.